unittests/test_cli.py

```diff
@@ -377,7 +377,7 @@ def test_performance_report(run_reframe):
         more_options=['-t', 'PerformanceFailureCheck', '--performance-report']
     )
     assert r'PERFORMANCE REPORT' in stdout
-    assert r'perf: 10 Gflop/s' in stdout
+    assert r'perf: 10 Gflop/s (50% of 20 Gflop/s)' in stdout
```

Comment:
Is the `20 Gflop/s` generic enough? 
Would be the same if we run on different systems?

Reply:
@victorusu I think this is the reference we set in `PerformanceFailureCheck` so it should be fine, right?